superbit_lensing/galsim/config_files/pipe_test.yaml

```diff
@@ -1,6 +1,6 @@
 # SuperBIT parameters
```

Comment:
Is there a `master_seed` missing here?

Reply:
No. `master_seed` is not a requirement, you can still pass in the various seeds explicitly as before. So I didn't bother to change the pipe test config.